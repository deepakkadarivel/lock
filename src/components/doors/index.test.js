import React from 'react';
import { create } from 'react-test-renderer';
import Doors from './index';

describe('Doors component', () => {
    test('Renders door component', () => {
        const component = create(<Doors />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});
