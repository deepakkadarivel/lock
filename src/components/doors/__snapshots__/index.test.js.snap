// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Doors component Renders door component 1`] = `
<h1>
  Doors
</h1>
`;
